Makefile

```diff
@@ -41,6 +41,12 @@ endif
 	docker-compose -f docker-compose.test.yml build --no-cache postman_test
 	docker-compose -f docker-compose.test.yml run --rm postman_test test/$(env).postman_environment.json --global-var "token=$(token)"
 
+test:
+	$(MAKE) unit-test
+	$(MAKE) integration-test
```

Comment:
what's the value of our integration-test in comparison to our postman tests? seems like postman is performing all the endpoints checks that the integration tests does prior.

Reply:
@tbellj Yup, agree.  Take a look at BCDA-712.  I'm working on that one next.